compiler/modules/nback.llvm/llvm.bal

```diff
@@ -87,6 +89,29 @@ public class Module {
         return fn;
     }
 
+    // Corresponds to LLVMGetIntrinsicDeclaration
+    public function getIntrinsicDeclaration(IntrinsicFunctionName name) returns FunctionDecl {
```

Comment:
The logic should add the intrinsic to the list of declarations only once, even if it is called multiple times. You will need a map or a table to manage this.

Reply:
Fixed